index.html

```diff
@@ -4,18 +4,39 @@
     <title>Tic Tac Toe!</title>
     <meta charset="utf-8">
     <link href="index.css" media="screen" rel="stylesheet" type="text/css">
+    <link href='https://fonts.googleapis.com/css?family=Quicksand:400,700' rel='stylesheet' type='text/css'>
+    <script src="sweetalert-master/dist/sweetalert.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="sweetalert-master/dist/sweetalert.css">
   </head>
 
   <body>
-    <h1>Tic Tac Toe</h1>
-    <div id="tic-tac-toe"></div>
+    <h1 class="header">Tic Tac Toe</h1>
+    <div id="board">
+      <table>
+        <tr id="row1">
+          <td class="square" id="r0c0"></td>
+          <td class="square v" id="r0c1"></td>
```

Comment:
Its better to be more descriptive when naming something. Like `v` for instance. It's hard to understand what you meant.